### ISSUE TITLE
Format of training data

### ISSUE THREAD

**Issue by alasokolova** [yjsunnn/DLoRAL#17]
Hey, thanks for your work, i tried it and it looks amazing!

I want also to try to train the model, but struggle with training data formatting.
Can you please clarify, what should be the format and the structure of the data, how it should be stored? And how it should be listed in `dataset_txt_paths` file?

Thanks in advance!

**Comment by alasokolova** [comment]
Thank you!
@yjsunnn one more question regarding training.
Is it possible to train a model not from scratch, but starting from existing checkpoint?
Seems that just ckpt is not enough and parameters config is also needed, right?

**Comment by yjsunnn** [comment]
I guess theoretically it's ok to start from the existing checkpoint, but from my perspective, it's not recommended for two reasons:

(1) The saved checkpoint only contains the model weights, but doesn't include the optimizer state (like momentum, learning rate schedules, etc.). Without these, the training dynamics would be quite different from a proper continuation.

(2) DLoRAL is actually already a fine-tuned model rather than trained from scratch. The model has already converged to a specific optimization landscape, so additional training on the same or similar data might lead to diminishing returns or even overfitting.

My own suggestions are:

(1) If you want to reproduce the training of DLoRAL, you can directly follow the README.md (sorry for missing data, it's just too large to upload. But all my data comes from [Pexel](https://huggingface.co/datasets/jovianzm/Pexels-400k), maybe you can download from the source).

(2) If you want to improve the existing performance of DLoRAL or create a totally new model to beat the performance of DLoRAL, I guess optimizing from my checkpoint will not be a good choice based on the second reason mentioned above. However, I think a more promising method is to use the iterative strategy of DLoRAL on a stronger backbone (like T2V models such as [Wan](https://github.com/Wan-Video/Wan2.2)). Existing VSR methods are still based on T2I models which highlight image quality and T2V models which obtains natural temporal consistency. Both approaches struggle to achieve an optimal balance between per-frame quality and temporal consistency. Therefore, adopting a stronger backbone would naturally lead to better performance than DLoRAL, while also presenting interesting research challenges regarding the inherent optimization dynamics and quality-consistency trade-offs in video super-resolution.

I hope this helps.

**Comment by alasokolova** [comment]
Thanks for suggestions!
And how long did it take you to train your model from scratch?
Training process seems to be quite slow, right?

**Comment by alasokolova** [comment]
got it, thanks!
One more question:
I noticed, that model checkpoint that you released was trained with lora rank = 16, while in your training config it is set to 4. Am I right?

**Comment by yjsunnn** [comment]
Yes, your observation is very sharp. We set it to 16 during training, but we find that actually 4 is enough for reproducing the same result. So we release the training config as rank=4.
